lib/api/core/plugins/pluginContext.js

```diff
@@ -135,19 +135,36 @@ class PluginContext {
         get: () => curryTrigger(kuzzle, pluginName)
       });
 
+      const getPluginSdk = sdk => ({
+        query: sdk.query.bind(sdk),
+        auth: sdk.auth,
+        bulk: sdk.bulk,
+        collection: sdk.collection,
+        document: sdk.document,
+        index: sdk.index,
+        ms: sdk.ms,
+        realtime: sdk.realtime,
+        security: sdk.security,
+        server: sdk.server
+      });
+
+      // TODO: Wait for the end of Node.js 6 support to use the spread operator
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)